packages/flutter/test/material/chip_test.dart

```diff
@@ -188,6 +188,93 @@ void _expectCheckmarkColor(Finder finder, Color color) {
   );
 }
 
+Future<void> pumpFrames({
+  WidgetTester tester,
+  int frames,
+  Duration duration,
+}) async {
+  int i;
+  for (i = 0; i < frames; i ++) {
+    await tester.pump(duration);
+  }
+}
+
+Widget _chipWithOptionalDeleteButton({
+  UniqueKey deleteButtonKey,
+  UniqueKey labelKey,
+  bool deletable,
+  TextDirection textDirection = TextDirection.ltr,
+}){
+  return _wrapForChip(
```

Comment:
Looks like the _wrapForChip helper takes a textDirection, so you can use that instead of adding a Directionality widget in here.